packages/contracts/scripts/deploy/dollar/solidityScripting/01_Diamond.s.sol

```diff
@@ -146,6 +151,28 @@ contract DiamondScript is Constants {
         IAccessControl = AccessControlFacet(address(diamond));
         StakingFacet IStakingFacet = StakingFacet(address(diamond));
         IStakingFacet.setBlockCountInAWeek(420);
+
+        // init UbiquityPool
+        // add collateral LUSD token
+        uint poolCeiling = 50_000e18; // max 50_000 of collateral tokens is allowed
```

Comment:
i think we could cast this to whole a `uint256`

Reply:
You mean to update to this code:
```solidity
 uint256 poolCeiling = 50_000e18;
```

?

Reply:
yes this is a big number let's cast it to a full uint256

Reply:
fixed https://github.com/ubiquity/ubiquity-dollar/pull/832/commits/acdd2754b6cf23d840abff312989c624528934e5